guides/v2.1/coding-standards/xml-coding-standard.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+group: coding-standards
+subgroup: Coding standards
+title: Layout XML Coding Standards
+menu_title: Layout XML Coding Standards
+menu_order: 100
+contributor_name: SwiftOtter Studios
+contributor_link: https://swiftotter.com/
+version: 2.1
+github_link: coding-standards/xml-coding-standard.md
+---
+
+# Layout XML Coding Standard
+
+### Name attribute
+
+The `name` attribute should adhere to the guidelines listed below. The `name` attribute should:
+
+- Be unique to the project.
+- Use a namespace approach where each child block's name contains its parent's name as well.
```

Comment:
Adding for the record (this is the salient point): 

> Use a namespace approach where each child block's name contains its parent's name

Because block names exist in the global layout scope, this isn't a "namespace approach"; rather, it's a hierarchy approach.